rmgpy/tools/generate_reactions.py

```diff
@@ -91,10 +91,11 @@ def execute(rmg, **kwargs):
     Returns an RMG object.
     """
     rmg.initialize(**kwargs)
-    
+
```

Comment:
Squash this in with the commit where you change enlarge

Reply:
Done